### ISSUE TITLE
Unable to resume a jupyter instance after session termination of the terminal.

### ISSUE THREAD

**Issue by rrkarim** [snarkai/snark-doc#2]
I can't find the appropriate feature in docs. Btw, I can stop and then start the instance again.

**Comment by davidbuniat** [comment]
Have you tried running ` snark attach pod_id --jupyter`?

**Comment by rrkarim** [comment]
Yes, it opens the connection on the new port.

**Comment by davidbuniat** [comment]
Right, thanks for noticing that! Will fix it. For now you can run 

 `snark attach pod_id -L 8888:localhost:8888`

Then go to localhost:8888
